desktop/src/@batch-flask/ui/info-box/info-box.component.ts

```diff
@@ -4,7 +4,7 @@ import { Component, Input } from "@angular/core";
     selector: "bl-info-box",
     template: `
         <div class="infobox">
-            <div class="infobox-image"><i class="fa fa-info-circle fa-3" aria-hidden="true"></i></div>
+            <div class="infobox-image"><i class="fa fa-info-circle fa-3" aria-hidden="true" title="Information"></i></div>
```

Comment:
This should be localized. That's a bit tricky given that it's in a hardcoded template in the Component annotation. My suggestion is to extract the template into its own html file, then also create a localized file (`info-box.i18n.yml`). The `info-box.module.ts` would need to import `I18nUIModule` and the spec file would also need to import `I18nUITestingModule` for the tests to succeed. Let me know if you need help with this.

Reply:
Good call! I thought I was doing something wrong because the raw localization string (the yaml path) was showing up in the UI and I got stuck there trying to figure out what's going on, but then I decided to restart electron and it worked!

Looks like new localization files are not picked up until I restart? or am i still doing something wrong? 😅

Anyways, this should be localized now:

<img width="269" alt="Screenshot 2023-06-02 at 3 30 25 PM" src="https://github.com/Azure/BatchExplorer/assets/79171711/f97f8914-a44c-4627-ac78-32f4318b781b">

Reply:
Yes, unfortunately, you have to restart Electron when a new file is introduced. But looks great. The only reason the build fails is that you need to run the translation build to update the strings file (`resources.resjson`). You can run `npm run build` from the root directory and commit the changes to the file.